contracts/ColonyNetworkAuction.sol

```diff
@@ -28,10 +28,14 @@ contract ColonyNetworkAuction is ColonyNetworkStorage {
   event AuctionCreated(address auction, address token, uint256 quantity);
 
   function startTokenAuction(address _token) public {
+    uint lastAuctionTimestamp = recentAuctions[_token];
+    require(lastAuctionTimestamp == 0 || now - lastAuctionTimestamp >= 30 days, "An auction may only be started once every 30 days.");
```

Comment:
We've been using kebab-case messages in requires. So something like `colony-auction-start-too-soon` would be appropriate here; the primary use case we're targeting is for clients to be able to interpret them with user-friendly messages, not for users to see them themselves.